static/src/js/components/DraftPreview/DraftPreview.jsx

```diff
@@ -198,6 +203,24 @@ const DraftPreview = () => {
     }
   }, [publishDraftLoading, publishDraftError])
 
+  const handleTemplate = async () => {
+    const response = await createTemplate(providerId, token, {
+      TemplateName: '',
+      ...ummMetadata
+    })
+
+    if (response.id) {
+      addNotification({
+        message: 'Collection template created successfully',
+        variant: 'success'
+      })
+
+      navigate(`/templates/collection/${response.id}`)
+    } else {
+      errorLogger('Error creating template', 'DraftPreview: handleTemplate')
```

Comment:
What is presented to the user in this case?

Reply:
Added a error notification.